.changes/header.tpl.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html),
+and is generated by [Changie](https://github.com/miniscruff/changie).
+
+Everything above this line (including this line) can be customized in `.changes/header.tpl.md`.
```

Comment:
@jerco - This is a template for what will appear at the top of the `CHANGELOG` - the file header as opposed to the change version header.  I'm going to look to you for what you want in the header if anything?  Right now it's nothing.  I like giving credit to changie but can see if you'd prefer it be somewhere else.

Reply:
Ah, sorry I missed this comment a few days ago. I do think it makes sense for the header to include a very simple explanation of what's going on here. Something like:

```
# dbt Core Changelog

- This file provides a full account of all changes to `dbt-core` and `dbt-postgres`
- Changes are listed under the (pre)release in which they first appear. Subsequent releases include changes from previous releases.
- "Breaking changes" listed under a version may require action from end users or external maintainers when upgrading to that version.
- Do not edit this file directly. This file is auto-generated using [Changie](https://github.com/miniscruff/changie). For details on how to document a change, see [the contributing guide](CONTRIBUTING.md)
```